### ISSUE TITLE
Allow labels for imported pub keys

### ISSUE THREAD

**Issue by sac001** [keys-pub/keys#31]
Hi,

it would be really nice if we had labels for public keys, so that we could give the keys names in order to know which key belongs to which person, once we have many of them in our key ring.

**Comment by gabriel** [comment]
Yeah I think being able to add labels and notes to keys could be helpful. Thanks for the feedback.
